AndroidSDKCore/src/main/java/com/leanplum/internal/Request.java

```diff
@@ -565,6 +568,10 @@ private void sendRequests() {
   }
 
   public void sendEventually() {
+    sendEventually(false);
+  }
+
+  public void sendEventually(boolean sendNow) {
```

Comment:
This argument is confusing. There's already a sendNow method. Methods should do 1 thing, which should be clear from the name of the method.